Medichost/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,42 @@
+import { NgModule } from "@angular/core";
+
+import { Routes, RouterModule } from "@angular/router";
+import { AppointmentDefaultComponent } from "./Appointment/appointment-default/appointment-default.component";
+import { SidebarComponent } from "./Appointment/sidebar/sidebar.component";
+import { HomeComponent } from "./Appointment/home/home.component";
+import { AppointmentsComponent } from "./Appointment/appointments/appointments.component";
+import { HomepageComponent } from "./Appointment/homepage/homepage.component";
+
+import { MedicoPatientsComponent } from "./Appointment/medico-patients/medico-patients.component";
+import { PatientsComponent } from "./Appointment/patients/patients.component";
+import { DashboardComponent } from './home/dashboard/dashboard.component';
+
+const routes: Routes = [
+    //{path: 'appointments', loadChildren: () => import('./appointment/appointment.module').then(m => m.AppointmentModule)},
+  // {path: 'doctor', loadChildren: () => import('./doctor/doctor.module').then(m => m.DoctorModule)},
+
+  {path:'auth', loadChildren: () => import('./auth/auth.module').then(m => m.AuthModule)},
+  { path: 'schedule', loadChildren: () => import('./scheduler/schedule.module').then(m => m.SchedulerModule)},
+  { path: 'patients', loadChildren: () => import('./scheduler/schedule.module').then(m => m.SchedulerModule) },
+  { path: 'patient', loadChildren: () => import('./patients/patients.module').then(m => m.PatientsModule)},
+  { path: 'error', loadChildren: () => import('./scheduler/schedule.module').then(m => m.SchedulerModule)},
+  { path: "**", component: DashboardComponent },
+
+
+ 
+  {
```

Comment:
This path won't be hit... due to the path with "**" above it

Try and swap their locations